kodilla-patterns2/src/test/java/com/kodilla/patterns2/observer/homework/MentorTestSuite.java

```diff
@@ -0,0 +1,29 @@
+package com.kodilla.patterns2.observer.homework;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MentorTestSuite {
+
+    @Test
+    public void testUpdate() {
+        //Given
+        TaskList taskListOne = new TaskListOne();
```

Comment:
Z nazewnictwa tych zmiennych powinno wynikać, że są to listy zadań różnych użytkowników (np. Cezarego Janickiego i Piotra Nowaka). Teraz są to po prostu dwie różne listy, ale z linii 20 i 24 wynika, że myślisz o nich jako o listach zadań z różnych modułów. Nie o to chodzi ;)
Jedna taskLista (jeden obiekt impl. `Observable`) = 1 kursant
Jeden `Observer`  = 1 mentor